### PR TITLE
Adapt getting started guides to new CLI

### PR DESCRIPTION
**Pull request by sophokles73**
This is (hopefully) the last step of #2914

### CODE REVIEW

File: site/documentation/content/getting-started/_index.md [hunk 1]
```diff
@@ -6,6 +6,567 @@ weight = 170
...
+~~~
+
+<a name="starting-a-consumer"></a>
+## Starting the example Application
+
+The telemetry data produced by devices is usually consumed by downstream applications that use it to implement their
+corresponding business functionality.
+In this guide we will use the Hono command line client to simulate such an application.
+The client will connect to the Kafka cluster that provides Hono's north bound Kafka based
+[Telemetry]({{< relref "/api/telemetry-kafka" >}}) and [Event API]({{< relref "/api/event-kafka" >}})s, subscribe
```
Comment (calohmn) [inline site/documentation/content/getting-started/_index.md:369]: The client will not necessarily connect to a Kafka cluster here.

File: site/documentation/content/getting-started/_index.md [hunk 2]
```diff
@@ -6,6 +6,567 @@ weight = 170
...
+
+{{% notice tip "Congratulations" %}}
+You have successfully connected a device to Hono and published sensor data for consumption by an application. The
+application consumed messages via Hono's north bound API regardless of the transport protocol used by the device to
+publish the data.
+
+**What to try next?**
+
+* Continue with the next sections to learn how applications can send commands to devices by means of the
+  [Command & Control API for Kafka]({{< relref "/api/command-and-control-kafka" >}}).
```
Comment (calohmn) [inline site/documentation/content/getting-started/_index.md:459]: AMQP should also be mentioned here.

File: site/documentation/content/getting-started/_index.md [hunk 3]
```diff
@@ -6,6 +6,567 @@ weight = 170
...
+
+The command line client will print an error message to *System.err* if the device does not respond to the command
+in time. By default, the client will wait for 60 seconds for the response, a longer timeout can be set using the `-r`
+option:
+
+~~~sh
+req --tenant ${MY_TENANT} --device ${MY_DEVICE} -n setBrightness --payload '{"level": 87}' -r 120
+~~~
+
+Congratulations. Now you have successfully sent commands to a device and responded to them.
```
Comment (calohmn) [inline site/documentation/content/getting-started/_index.md:567]: Putting this chapter in a green `Congratulations` box like above would be nice.

**Comment by sophokles73** [comment]
@calohmn I have pushed changes

**Review by calohmn — APPROVED** [review]
LGTM

**Pull request merged**